### ISSUE TITLE
Village Fuel and Village Buckets are overbearing requirements.

### ISSUE THREAD

**Issue by JohnyDL** [TangoTek/TekTopia-Community#628]
Some things seem frustrating when it comes what's required to make or be village items. 

TL;dr: Fuels and Buckets shouldn't be required to be village items. To cook village items or gather village milk

I'll go into excruciating detail below but I feel like fuels and buckets needing to be village items is a lot like requiring Picks, Hoes and Axess to be village items to get village ores, crops, logs and butcher drops. And it can cause as much trouble.

---------

None village charcoal will be used to cook village wood into more none village charcoal because villagers want to use up the none village items as soon as possible but in this case, end up multiplying them.

None village charcoal will be used then to cook village ores into none village ingots or food ruining the value of those items by killing durability or removing from the economy entirely.

Heat is heat, it shouldn't matter if the heat is provided by Lava buckets, village collected resources or player collected resources. And in the case of the infinite none village charcoal loop it's a spiral that can kill your economy if you don't actively put an end to it.

When eventually you acquire village buckets, one used on a none village cows makes none village milk buckets and making the bucket permanently useless for village cake for no good reason. The bucket has no value to the economy or the village in any tangible way, it's not even a real ingredient for cake nor is it sellable. It's only the milk that has any value.

Village cake isn't an economy thing but it is a thing that makes the village happier and provides a lot of nutrition, halving those values is kinda annoying, all for the sake of the rancher picking the wrong bucket/cow combination.

But to take this back to the TL;dr Farmers, Lumberjacks, Miners, and Butchers don't require village tools to do their jobs, in fact, we don't even care about the origin of the materials they collect in 3/4 cases (not that we can except village animals) just so long as the villager is the one who harvests the blocks/items not the player they will be village items.

Empty buckets and fuels, in my opinion, are just tools used to convey or convert other items, and we should only care if the milk came from a village cow or the item being cooked in the furnace was a village item before they were cooked.

I know this is probably working as intended but I wanted to have my frustrated say :p
